assets/dev/scss/frontend/widgets/button.scss

```diff
@@ -0,0 +1,48 @@
+.elementor-widget-button {
```

Comment:
Move your styling to the existing file: `assets/dev/scss/frontend/general/_button.scss`